app/views/pxe/_pxe_image_type_form.html.haml

```diff
@@ -22,3 +22,5 @@
     :javascript
        miqInitSelectPicker();
        miqSelectPickerEvent("provision_type", "#{url}")
+
+= react 'PxeImageForm', :recordId => @pxe_image_type.id.to_s
```

Comment:
Will this work if you're not editing but creating a new entity?

Reply:
It does the opposite right now, only renders when creating a new entity not when editing

Reply:
That's because you're not including the [API change](https://github.com/ManageIQ/manageiq-api/issues/908) I linked in the [issue](https://github.com/ManageIQ/manageiq-ui-classic/issues/7392#issuecomment-764450671). I think it should be also in the PR description.

Also I'd rather play this safely by checking if we're editing or not:
```ruby
@pxe_image_type.try(:id).try(:to_s)
```